src/Components/test/testassets/TestServer/ProtectedBrowserStorageStartup.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace TestServer
+{
+    public class ProtectedBrowserStorageStartup
+    {
+        public ProtectedBrowserStorageStartup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
```

Comment:
```suggestion
            services.AddRazorPages();
```

Does this work? We tell everyone to prefer this to AddMvc